research/README.md

```diff
@@ -0,0 +1,39 @@
+# Research
+
+This folder holds ongoing research and topics of interest. The [Interchain Foundation](https://interchain.io) has research topics that they are willing fund the research for, to find out more you can go to there [funding repo](https://github.com/interchainio/funding/blob/master/research.md). You can apply for a research grant [here](https://docs.google.com/forms/d/e/1FAIpQLSclH4R5G7WgNpKPvXxPPFRA7rAoyX8nNvsJAQJpZNZwWWjFmA/viewform)
+
+## Areas of Interest
+
+Below you will find some areas of interest which we have.
```

Comment:
I would suggest removing the text below as I am not sure that this list reflects in the best way short/midterm topics of interest. We haven't updated it for quite some time, but maybe it's safer to remove it for this PR and we can maybe update it in a separate PR.